app/assets/v2/js/grants/fund.js

```diff
@@ -83,6 +85,33 @@ $(document).ready(function() {
             $('#sub_new_approve_tx_id').val(transactionHash);
             const linkURL = etherscan_tx_url(transactionHash);
 
+            let data = {
```

Comment:
if `$form` is your form, you can just use `$form.data()` and avoid this big dict creation i think!

Reply:
The only thing is that I dont want all data in the form to be passed through. That way I can do `if` statements in the view based on the content of the `POST`. If there is a better workaround for that happy to implement it!